terraform/layers/python3.7/fastapi_0.65.2/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM lambci/lambda:build-python3.7
```

Comment:
```suggestion
FROM lambci/lambda:python3.8
```

Reply:
The `build` image has all the dependencies to compile source code and is much larger. I don't believe you need all of that here. But if yes, I would go with the [multi stage approach](https://docs.docker.com/develop/develop-images/multistage-build/) and use the lighter image as the final image.

I would also use a newer python version. It doesn't look like they already have a python 3.9 image but 3.8 is available

Reply:
Thanks! I initially tried 3.8 but had trouble. I don't remember what it was, though, so I guess I'll try it again and find out. I'll also try switching to the smaller, non-build, image, because why not? Might shave a few seconds off.

Reply:
Ran into some permissions trouble with the non-build image, but I did successfully update from Python 3.7 to 3.8.